templates/user_data.elasticsearch.sh

```diff
@@ -12,14 +12,16 @@ node.data: ${data}
 node.ingest: ${data}
 xpack.security.enabled: ${security_enabled}
 xpack.monitoring.enabled: ${monitoring_enabled}
+xpack.license.self_generated.type: basic
```

Comment:
make it a variable
```suggestion
xpack.license.self_generated.type: ${xpack_license_self_generated_type}
```

Or just remove if not really needed

Reply:
There was some issues with the licensing, but I didn't spend time looking into it. Hard coding the license as the free, basic type resolved the issues for now